conf/test_contamination.config

```diff
@@ -0,0 +1,40 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/smrnaseq -profile test_contamination,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function with contamination filter'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+
+    input            = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/samplesheet/v2.0/samplesheet.csv'
+    fasta            = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/genome.fa'
+
+    mirtrace_species         = 'hsa'
+    skip_mirdeep             = true
+    save_merged              = false
+
+
+    filter_contamination = true
+    cdna                 = "https://huggingface.co/datasets/nf-core/smrnaseq/resolve/main/GRCh37/Homo_sapiens.GRCh37.cdna.all.fa"
+    ncrna                = "https://huggingface.co/datasets/nf-core/smrnaseq/resolve/main/GRCh37/Homo_sapiens.GRCh37.ncrna.fa"
+    trna                 = "https://huggingface.co/datasets/nf-core/smrnaseq/resolve/main/GRCh37/hg19-tRNAs.fa"
```

Comment:
I am not familiar with huggingface.co.  Would it be possible to have all the test data on GitHub?

Reply:
Unfortunately, no. These are too large to be direclty uploaded to github (and our testdata repository) so this wa saligned with the core team to use it lik ethis. 

Thanks for the review @charles-plessy :)

Reply:
Unfortunately, if I recall correctly, it was difficult to upload these files to GitHub due to size, but let's confirm with @nschcolnicov